src/server.cpp

```diff
@@ -1395,7 +1395,7 @@ void CServer::CreateAndSendChatTextForAllConChannels ( const int      iCurChanID
         "<font color=""" + sCurColor + """>(" +
         QTime::currentTime().toString ( "hh:mm:ss AP" ) + ") <b>" +
         ChanName.toHtmlEscaped() +
-        "</b></font> " + strChatText;
+        "</b></font> " + strChatText.toHtmlEscaped();
```

Comment:
I assume it works as intended. Should I test it again?

Reply:
I think the change is trivial enough just to approve. I tested it:
![image](https://user-images.githubusercontent.com/3224952/107353795-5c070e80-6ac5-11eb-8264-ec2728295511.png)